core/src/main/kotlin/pages/PageBuilder.kt

```diff
@@ -16,11 +17,13 @@ class DefaultPageBuilder(
             p.classes.map { pageForClass(it) } +
                     p.functions.map { pageForMember(it) })
 
-    override fun pageForClass(c: Class): ClassPageNode =
-        ClassPageNode(c.name, contentForClass(c), c.dri, c,
-            c.constructors.map { pageForMember(it) } +
+    override fun pageForClass(c: ClassLike): ClassPageNode {
+        val constructors = if (c is Class) c.constructors else emptyList()
+        return ClassPageNode(c.name, contentForClasslike(c), c.dri, c,
```

Comment:
Again, `pageForClassLike` should only dispatch creating appropriate `PageNode` to some other functions, namely `pageForClass` and `pageForEnum`